synapse/models/infotech.py

```diff
@@ -10,14 +13,14 @@ def getBaseModels():
                 ('it:hostname', {'subof': 'str:lwr', 'doc': 'A system/host name'}),
 
                 ('it:hosturl', {'subof': 'comp', 'fields': 'host,it:host|url,inet:url'}),
-                ('it:hostfile', {'subof': 'comp', 'fields': 'host,it:host|path,file:path|file,file:bytes',
-                                 'doc': 'A file created on a host'}),
+                ('it:hostfile', {'subof': 'comp', 'fields': 'host=it:host', 'optfields':'path=file:path,file=file:bytes,time=time', 'doc': 'A file created on a host'}),
```

Comment:
Probably created.

If the file was modified but unchanged (open file, save file, modified timestamp changes), I don't think anything really analytically relevant occurred. If the file was modified but changed, it would have a new file:bytes, which would create a different node.

Having "created time" in the comp type allows us to at least track if the "same" file has been created at the same path on the same host multiple times.